action.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Run quickstart'
+description: 'Runs quickstart docker image'
+inputs:
+  version:
+    description: "Version (docker tag) of quickstart image to use"
+    required: true
+    default: "latest"
+runs:
+  using: "composite"
+  steps:
+    - name: Setup docker (missing on MacOS)
+      shell: bash
+      if: runner.os == 'macos'
+      run: |
+        brew install docker
+        brew install colima
+        colima start
+    - run: |
+        docker run -d --name quickstart -p 8000:8000 -e ENABLE_LOGS=true -e NETWORK=local -e ENABLE_SOROBAN_RPC=true --health-cmd "curl --no-progress-meter --fail-with-body -X POST \"http://localhost:8000/soroban/rpc\" -H 'Content-Type: application/json' -d '{\"jsonrpc\":\"2.0\",\"id\":8675309,\"method\":\"getNetwork\"}' && curl --no-progress-meter \"http://localhost:8000/friendbot\" | grep '\"invalid_field\": \"addr\"'" --health-interval 10s --health-timeout 5s --health-retries 50 stellar/quickstart:${{ inputs.version }}
```

Comment:
All the configurations options like logs and network should be inputs on the action with these defaults. So that folks can use it with other options.

Or, if we can pass through ENV vars, we should probably do that and only set these env vars if none are set. Wdyt?

Reply:
Added options as inputs